NEXTCLOUD.cmake

```diff
@@ -1,17 +1,28 @@
-set( APPLICATION_NAME       "Nextcloud" )
-set( APPLICATION_SHORTNAME  "Nextcloud" )
-set( APPLICATION_EXECUTABLE "nextcloud" )
+if(NEXTCLOUD_DEV)
+    set( APPLICATION_NAME       "NextcloudDev" )
+    set( APPLICATION_SHORTNAME  "NextDev" )
+    set( APPLICATION_EXECUTABLE "nextclouddev" )
+    set( APPLICATION_ICON_NAME  "Nextcloud" )
+else()
+    set( APPLICATION_NAME       "Nextcloud" )
+    set( APPLICATION_SHORTNAME  "Nextcloud" )
+    set( APPLICATION_EXECUTABLE "nextcloud" )
+    set( APPLICATION_ICON_NAME  "${APPLICATION_SHORTNAME}" )
+endif()
+
 set( APPLICATION_CONFIG_NAME "${APPLICATION_EXECUTABLE}" )
 set( APPLICATION_DOMAIN     "nextcloud.com" )
 set( APPLICATION_VENDOR     "Nextcloud GmbH" )
 set( APPLICATION_UPDATE_URL "https://updates.nextcloud.org/client/" CACHE STRING "URL for updater" )
 set( APPLICATION_HELP_URL   "" CACHE STRING "URL for the help menu" )
 
 if(APPLE AND APPLICATION_NAME STREQUAL "Nextcloud" AND EXISTS "${CMAKE_SOURCE_DIR}/theme/colored/Nextcloud-macOS-icon.svg")
-    set( APPLICATION_ICON_NAME "Nextcloud-macOS" )
+    if(NEXTCLOUD_DEV)
+        set( APPLICATION_ICON_NAME "Nextcloud-macOS-dev" )
+    elseif()
+        set( APPLICATION_ICON_NAME "Nextcloud-macOS" )
+    endif()
     message("Using macOS-specific application icon: ${APPLICATION_ICON_NAME}")
-else()
-    set( APPLICATION_ICON_NAME "${APPLICATION_SHORTNAME}" )
```

Comment:
The release icon now works ok, but, why do we need to have a different icon name for the dev version at all? The dev version will have a broken icon as there is no Nextcloud-macOS-dev.svg file